packages/core/src/Stores/notification-store.js

```diff
@@ -205,7 +206,11 @@ export default class NotificationStore extends BaseStore {
             id,
             buy_price,
             contract_id,
-            contract_type: getTradeTypeName(contract_type, isHighLow({ shortcode }), isTurbosContract(contract_type)),
+            contract_type: `${getContractBasis(contract_type)} ${getTradeTypeName(
```

Comment:
According to the requirements, we should show notification as `[contract type][contract subtype]` (e.g. Turbos Up) for Multipliers, Vanillas and Turbos. `getContractBasis()` will return basis ([contract type]) for mentioned contracts and empty string for the rest